actions/chats.js

```diff
@@ -53,6 +65,20 @@ export const fetchChats = (data) => {
     };
 };
 
+export const fetchMessagesInChat = (cid) => {
+    return async (dispatch, getState) => {
+        const headers = await addAuthHeader();
+        const requestString = `${baseUrl}chat/` + cid;
```

Comment:
minor thing, the `` syntax makes it so you can mix strings and variables, so you can do ``${baseUrl}chat/${cid}``

Reply:
Ah great, I was thinking that at first but somehow didn't think it was working when it was actually something else :).